### PR TITLE
Remove travis build status from README

### PR DESCRIPTION
**Pull request by GSadee**

### CODE REVIEW

**Review by loic425 — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SyliusGridBundle [![Build status...](https://secure.travis-ci.org/Sylius/SyliusGridBundle.png?branch=master)](http://travis-ci.org/Sylius/SyliusGridBundle)
+SyliusGridBundle
```
Comment (lchrusciel) [inline README.md:1]: But would be perfect to replace it with github actions badge
```suggestion
SyliusGridBundle [![Build](https://github.com/Sylius/SyliusGridBundle/actions/workflows/build.yml/badge.svg)](https://github.com/Sylius/SyliusGridBundle/actions/workflows/build.yml)
```

Comment (loic425) [reply README.md:1]: @lchrusciel You're right, that's even better.

Comment (GSadee) [reply README.md]: I've added the badge 👌🏻

**Review by loic425 — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, Grzegorz! :1st_place_medal:

**Pull request merged**